src/applications/facility-locator/actions/index.js

```diff
@@ -15,22 +14,10 @@ import {
 import LocatorApi from '../api';
 import { LocationType, BOUNDING_RADIUS } from '../constants';
 import { ccLocatorEnabled } from '../config';
-import environments from '../../../platform/utilities/environment';
 
-/**
- New: mbxClient is the new instance for the API calls and
- The new SDK requires to pass the mapbox client to geocode services
- otherwise use the client as it is
-*/
-let mbxClient;
-
-if (environments.isStaging()) {
-  const mbxGeo = require('@mapbox/mapbox-sdk/services/geocoding');
-  mbxClient = mbxGeo(mapboxClient);
-} else {
-  mbxClient = mapboxClient;
-}
+const mbxGeo = require('@mapbox/mapbox-sdk/services/geocoding');
```

Comment:
Same question as @kelsonic asked in VAMap.jsx: Can this be an `import` statement?